pkg/set/set.go

```diff
@@ -0,0 +1,15 @@
+package set
+
+import (
+	"context"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+)
+
+func SetProfile(ctx context.Context, clientsets *k8sutil.Clientsets, recoveryOption, operatorNamespace, storageClusterNamespace string) {
+
+	cephArgs := []string{"config", "set", "osd", "osd_mclock_profile", recoveryOption}
+
+	exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", cephArgs, operatorNamespace, storageClusterNamespace, true, false)
```

Comment:
We have a recent Rook design doc that intends allow users to specify Ceph config changes on the CephCluster itself. If that feature were available in Rook, would the CLI:
1. deprecate this command and tell users to use the CephCluster config spec?
2. modify this command to use the CephCluster config spec?
3. do neither?

Do we need to consider the answer to this question before proceeding here?

See: https://github.com/rook/rook/pull/13202

Reply:
I see, but currently, that is in cephCluster cr not in storageCluster cr which is what ocs operator will reconcile. Also, the main reason for only individual ceph commands is because we discussed that we don't want to expose all the ceph options to customers only add what they request for or want.

Also, if in the future we add in storageCluster cr to allow modifying all the ceph config then in this command we can update the command to patch storageCluster cr but this will require discussion with the support team also what the customer wants as currently customer requests was a tool like this as they were not interested in running the commands in the toolbox.

Reply:
Also, somehow I feel the rook feature to modify the ceph config should be in configmap similar to `rook-ceph-operator-config` configmap where we add operator and CSI settings rather than in cephCluster CR.  IMO we'll make cephCluster CR too big and complicated.